### PR TITLE
Confirmed Tx UI

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
image

### CODE REVIEW

File: src/pages/TxSender.vue [hunk 1]
```diff
@@ -41,11 +41,30 @@
...
 				>
 			</a-col>
 		</a-row>
+		<a-row v-if="confirmedResponse" class="margin_top_med">
+			<div>
+				<h1>Transaction Receipt</h1>
+				<div>
+					<h4>
+						<a href="https://algoexplorer.io/" target="_blank"
+							>Open Algo Explorer</a
```
Comment (robert-zaremba) [inline src/pages/TxSender.vue]:
```suggestion
							>Open in Algo Explorer</a
```

File: src/pages/TxSender.vue [hunk 2]
```diff
@@ -41,11 +41,30 @@
...
 					>Send</a-button
 				>
 			</a-col>
 		</a-row>
+		<a-row v-if="confirmedResponse" class="margin_top_med">
+			<div>
+				<h1>Transaction Receipt</h1>
+				<div>
+					<h4>
+						<a href="https://algoexplorer.io/" target="_blank"
```
Comment (robert-zaremba) [inline src/pages/TxSender.vue]: you need to construct the right link with tx id.

**Review by robert-zaremba — COMMENTED** [review]
- clicking on "open in algo explorer" should directly open the transaction

File: src/pages/TxSender.vue (hunk 2 above)
Comment (Megha-Dev-19) [reply src/pages/TxSender.vue]: okay will update the link

File: src/pages/TxSender.vue [hunk 3]
```diff
@@ -41,11 +41,28 @@
...
 					@click="openConfirmationModal"
-					:disabled="!txOutput"
+					:disabled="!txOutput || isSendDisabled"
 					>Send</a-button
 				>
 			</a-col>
 		</a-row>
+		<a-row v-if="confirmedResponse" class="margin_top_med">
+			<div>
+				<h1>Transaction Receipt</h1>
```
Comment (robert-zaremba) [inline src/pages/TxSender.vue]: h1 should be used for page title, here, for subsections we want probably h2 or h3

File: src/pages/TxSender.vue [hunk 4]
```diff
@@ -41,11 +41,28 @@
...
+				<div>
+					<h4>
+						<a :href="algoExplorerURl" target="_blank">Open in Algo Explorer</a>
+					</h4>
```
Comment (robert-zaremba) [inline src/pages/TxSender.vue]: Is this a header? Probably not.

File: src/pages/TxSender.vue [hunk 5]
```diff
@@ -134,6 +155,17 @@ export default defineComponent({
...
 			}
+			let txID = "";
+			if (!this.isJsonSelected) {
+				const decodedTxn = decodeObj(
+					convertBase64ToUnit8Array(this.txInput)
+				) as EncodedSignedTransaction;
+
+				txID = algosdk.Transaction.from_obj_for_encoding(decodedTxn.txn).txID();
+			} else {
+				// TODO : ADD TxID support for json
```
Comment (robert-zaremba) [inline src/pages/TxSender.vue:168]: please create a task for this in pivotal tracker

File: src/store/WalletStore.ts [hunk 6]
```diff
@@ -34,6 +35,22 @@ export default defineStore("WalletStore", {
...
+				case NetworkTypes.MAIN_NET:
+					this.algoExplorerURL =
+						"https://algoindexer.algoexplorerapi.io/v2/transactions/";
+					break;
+				case NetworkTypes.TEST_NET:
+					this.algoExplorerURL =
+						"https://algoindexer.testnet.algoexplorerapi.io/v2/transactions/";
+					break;
+			}
+		},
```
Comment (robert-zaremba) [inline src/store/WalletStore.ts:51]: I thought we already have it in our wallet widget.

**Review by robert-zaremba — APPROVED** [review]
pre approving

File: src/store/WalletStore.ts (hunk 6 above)
Comment (Megha-Dev-19) [reply src/store/WalletStore.ts:51]: we have the url in web library and wallet widget for node not for indexer

File: src/pages/TxSender.vue (hunk 5 above)
Comment (Megha-Dev-19) [reply src/pages/TxSender.vue:168]: okay

File: src/pages/TxSender.vue (hunk 3 above)
Comment (Megha-Dev-19) [reply src/pages/TxSender.vue]: updated

File: src/pages/TxSender.vue (hunk 4 above)
Comment (Megha-Dev-19) [reply src/pages/TxSender.vue]: updated

**Pull request merged**